modules/squants/shared/src/main/scala/io/circe/squants/package.scala

```diff
@@ -0,0 +1,99 @@
+package io.circe
+
+import _root_.squants._
+import _root_.squants.Quantity
+
+import _root_.squants.experimental.formatter.syntax._
+import _root_.squants.experimental.formatter._
+import _root_.squants.information.{Information, Megabytes}
+import _root_.squants.information.InformationConversions._
+import _root_.squants.experimental.unitgroups.information.{IECInformation, MetricInformation}
+
+import scala.reflect.runtime.universe._
+import scala.util.{Failure, Success, Try}
+
+import scala.collection.Set
+
+
+/**
+  * Provides codecs for [[https://github.com/typelevel/squants squants]] types.
+  *
+  * @author Quentin ADAM @waxzce
+  */
+
+
+package object squants {
+
+  def getCompagnonObject[A <: Quantity[A]](implicit man: TypeTag[A]) = {
```

Comment:
`s/Compagnon/Companion/`

Reply:
Does this need to be public? If it absolutely must be, can you add a type annotation?

Reply:
yep, i'll update it no need to be public indeed